pkgs/games/dwarf-fortress/themes/default.nix

```diff
@@ -1,4 +1,4 @@
-{lib, fetchFromGitHub}:
+{lib, fetchFromGitHub, ...}:
```

Comment:
Why the `...`

Reply:
Not sure, was there previously. Can remove it.